.circleci/config.yml

```diff
@@ -13,11 +25,11 @@ jobs:
           name: Install prerequisites
           command: |
             sudo apt-get update
-            sudo apt-get install -y libsasl2-dev python3.8-venv python3.8-dev
+            sudo apt-get install -y libsasl2-dev
       - run:
           name: Setup environment
           environment:
-            VENV_BIN: "/usr/bin/python3.8 -m venv"
+            VENV_BIN: "python3 -m venv"
```

Comment:
we can get rid of this now since it is the default value in the makefile :-)

Reply:
Done, leftover from debugging :)